File: fix-store.js
const fs = require('fs');
const path = 'frontend/app/store/page.tsx';
let s = fs.readFileSync(path, 'utf8');
// Fix: remove '</div>' + curly apostrophe + spaces + ')}' and fix structure
s = s.replace(/<\/div>[\u2019'\s]+\)\}/g, '</div>\n                  </main>\n                </>\n              )}');
fs.writeFileSync(path, s);
console.log('Done');

File: fix-store.py
import re
with open('frontend/app/store/page.tsx', 'r', encoding='utf-8') as f:
    s = f.read()
s = re.sub(r"\)\}\s*they['']ll\s+</div>", ')}\n                    </div>', s)
with open('frontend/app/store/page.tsx', 'w', encoding='utf-8') as f:
    f.write(s)
print('Done')
